module2/homework/insensitivePalindrom/palindrome.cpp

```diff
@@ -0,0 +1,16 @@
+#include "palindrome.hpp"
+
+#include <algorithm>
+
+bool is_palindrome (const std::string& expression)
+{
+    std::string tmp;
+    std::copy_if(expression.begin(), expression.end(),
+            back_inserter(tmp), [](auto x) { return std::isalpha(x); });
```

Comment:
Is lambda required here. Isn;t isalpha a functor?

Reply:
+1

Reply:
cpp reference calls this function inside lambda too.  https://en.cppreference.com/w/cpp/string/byte/isalpha